.github/workflows/main.yaml

```diff
@@ -29,8 +29,19 @@ jobs:
             echo "Defaulting to todays date ${releaseDate}"
           fi
 
+          echo "releaseDate=${releaseDate}" >> $GITHUB_ENV
+
           dotnet run --project build/tools/Automation/Automation.csproj ${releaseDate} ${{ github.workspace }}
 
+      - name: Check if any changes exist.
```

Comment:
nit: would you mind updating this text to be a little more specific as to what changes you're checking for? (in this case, it's Git changes, but I could see it being confused for SDK changes at the package manager level)

Reply:
Updated. Hm do you have a more specific scenario of when this will behave unexpectedly?